### PR TITLE
[#159193487] add new screen to display when the user has not SPID

### PR DESCRIPTION
**Pull request by valentinacala**
This pr introduce the screen presented below. Selecting the footer button the user is redirected on the browser (https://www.spid.gov.it/)

image

image

Does this screen substitute the previous SpidInformationRequestScreen ?
Is the image available?
image

### CODE REVIEW

File: ts/screens/authentication/SpidInformationScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+        
+                <AppHeader>
+                  <Left>
+                    <Button transparent={true} onPress={_ => this.goBack()}>
+                      <IconFont name="io-back" />
+                    </Button>
+                  </Left>
+                  <Body>
+                    <Text>
+                      {I18n.t("authentication.spid_information_request.headerTitle")}
```
Comment (fpersico) [inline ts/screens/authentication/SpidInformationScreen.tsx]: I think authentication.spid_information_request.headerTitle must be changed

File: ts/theme/variables.ts [hunk 2]
```diff
@@ -131,6 +134,16 @@ const customVariables = Object.assign(materialVariables, {
     return this.fontSize6 * this.lineHeightFontSizeRatio;
   },
 
+   //H2
```
Comment (cloudify) [inline ts/theme/variables.ts]: @fpersico is this ok?

Comment (valentinacala) [reply ts/theme/variables.ts]: I implemented H2 to render the numbered list being it not used (I did not exist)

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Remember to fix the PR title by removing WIP wording when ready
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#159193487](https://www.pivotaltracker.com/story/show/159193487): Come CIT se non ho spid voglio vedere una pagina di contenuto testuale con un CTA a spid.gov.it

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by valentinacala** [comment]
New commit introduce the DefaultSubscreenHeader to display both the sub-header title and the image. comparing with the prototype for this and other screens, this component should be updated so that:
- the bottom line of the title is aligned with  the image bottom line
- the image is presented with the proper dimension.
Here how the screen actually appear:
image


here how it should be:
image

**Pull request merged**